tests/Makefile

```diff
@@ -37,7 +37,7 @@ run:
 		-it -v ${LOCAL_ARTIFACT_DIR}/:$(ARTIFACT_DIR):z -v /tmp/tests-kubeconfig:/tmp/kubeconfig:z $(IMAGE)
 
 clean:
-	oc delete -n $(ODHPROJECT) kfdef opendatahub || true
+	oc delete -n $(ODHPROJECT) kfdef odh-core || true
```

Comment:
@anishasthana does this need to be changed back to `opendatahub`? I know we made some temporary edits to this file while testing.

Reply:
This is actually an error that I fixed -- the kfdef name is odh-core now.